packages/gatsby-recipes/src/renderer/render.js

```diff
@@ -1,6 +1,7 @@
 import React, { Suspense, useContext, useState } from "react"
-import Queue from "p-queue"
+import Queue from "better-queue"
```

Comment:
was having odd problems with p-queue not providing events when things finished so moved to the tried & true better-queue